src/index.html

```diff
@@ -11,7 +11,163 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    autocomplete="off">
+
+      <fieldset class="fieldset">
```

Comment:
Don't add empty lines between parent and child elements.
```suggestion
    autocomplete="off">
      <fieldset class="fieldset">
```